proxies/live/apiproxy/resources/jsc/validate-claim.js

```diff
@@ -0,0 +1,13 @@
+jwksString = context.getVariable('jwks');
+jwksObj = JSON.parse(jwksString);
+kid = context.getVariable('jwt.DecodeJWT.FromSubjectTokenFormParam.decoded.header.kid');
+
+token_updated='false';

```

Comment:
The variable should have a more accurate name like jwks_updated because it's not the jwt that needs updating